symengine/tests/basic/test_sets.cpp

```diff
@@ -480,7 +493,7 @@ TEST_CASE("Complement : Basic", "[basic]")
     REQUIRE(eq(*r4.get_universe(), *i1));
 
     REQUIRE(r1->get_args().size() == 2);
-    REQUIRE(eq(*r1->contains(one), *boolTrue));
+    REQUIRE(is_a<Not>(*r1->contains(one)));
```

Comment:
at this point, `r1` stores `(-oo, oo) \ {y} `. But here we are not sure if `finiteset({y})` contains `one` or not.
So, this should return `not(contains(1,{y}))`. Earlier this bug was overshadowed by the bug in FintieSet `contains`, which I corrected now.